calico_containers/calico_ctl/bgp.py

```diff
@@ -56,8 +56,12 @@ def validate_arguments(arguments):
     asnum_ok = True
     if arguments.get("<AS_NUM>"):
         try:
-            asnum = int(arguments["<AS_NUM>"])
-            asnum_ok = 0 <= asnum <= 4294967295
+            asnum = str(arguments.get("<AS_NUM>"))
```

Comment:
Can this get return None?  In that case str(None) will return a string "None".

I don't think you want the str()